rpa-manager/modules/ROOT/pages/processautomation-prepare.adoc

```diff
@@ -258,51 +285,93 @@ The credential can be deployed in the production phase.
 
 * *Type*:
 +
-Type of the global credential. The type of credential determines where it can be used. The original value defined in RPA Builder is typed. Select one of the following types:
+Type of the global credential. The type of credential determines where you can use it. The original value defined in RPA Builder is typed. Select one of the following types:
 +
 ** *AWS Credentials*:
 +
-The credential is used for accessing Amazon Web Services. Fill in the client ID, client secret and region. Using a session token is optional.
+Use this credential for accessing Amazon Web Services. Fill in the client ID, client secret and region. Using a session token is optional.
+** *OAuth Credentials*
++
+Use this credential for accessing the Microsoft Outlook email application via Microsoft Azure.
++
+*** *OAuth Host*:
++
+The host address used to get the credentials. The default address for Outlook services is https://login.microsoftonline.com/common/oauth2/v2.0/ . This address might change if the customer uses a self hosted service.
+*** *Client ID*:
++
+To authenticate with the OAuth Host, an Azure application has to be defined in the customers space. This Azure application has an id, that is unique in the entire OAuth Host space and allows a user to explicitly grant or revoke access for this Azure application to his account.
++
+For more information how to create this Azure application for Outlook, see https://learn.microsoft.com/en-us/azure/active-directory/develop/quickstart-register-app[Quickstart: Register an application with the Microsoft identity platform^]
+*** *Client Secret*
++
+This secret allows RPA Manager to prove to the OAuth Host, that it has received the permission to access user accounts on behalf of the registered Azure application. Without it the authentication is not possible.
+*** *Redirect URI*
++
+This is the URI to which the OAuth Host will redirect after the user has completed the authentication attempt. This redirect URI has to be registered with the Azure application. Only if the entered redirect URI matches one of the registered URI, the OAuth Host will allow the authentication.
+*** *Scopes*
++
+The scopes define, which permission the user grants to the Azure application. By default, we request the following permissions:
```

Comment:
```suggestion
The scopes define which permission the user grants to the Azure application. By default, we request the following permissions:
```

Reply:
Can you change "we" to "RPA"?

Reply:
Replaced with 'Grant the following permissions:'